packages/grafana-ui/src/components/Drawer/Drawer.tsx

```diff
@@ -121,7 +122,8 @@ export function Drawer({
                   variant="secondary"
                   fill="text"
                   onClick={onClose}
-                  aria-label={selectors.components.Drawer.General.close}
+                  data-testid={selectors.components.Drawer.General.close}
+                  aria-label={t('selectors.components.Drawer.General.close', 'Close Drawer')}
```

Comment:
```suggestion
                  aria-label={t('grafana-ui.drawer.close', 'Close')}
```

And be sure to run `yarn i18n:extract` and commit the results :)

Reply:
Hi @joshhunt  Changes implemented